easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+#
+easyblock = 'CMakeMake'
+name = 'Brotli'
+version = '1.0.9'
+
+homepage = 'https://github.com/google/brotli'
+description = """Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination
+ of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio
+ comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate
+ but offers more dense compression.
+The specification of the Brotli Compressed Data Format is defined in RFC 7932."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
```

Comment:
@fizwit other compression software are built with `GCCcore` toolchain, I think we should go with that here as well?

Reply:
Brotli was required by some Python package, required by some bioinformatics thing. I'm so deep into the dependencies I forgot what it was.

Reply:
but does not matter, right? if the GCC-based version can be used as dep, GCCcore-based should work too.